DuckDuckGo/Localizable.xcstrings

```diff
@@ -35109,6 +35109,18 @@
         }
       }
     },
+    "newTab.menu.item.show.search.bar" : {
```

Comment:
❓ Were these translations uploaded to Smartling?

Reply:
No, they weren't. I'll have one more small-ish PR with another piece of copy and will request translation once the copy review is done. I will only merge it when we have translations anyway.